### PR TITLE
Self destroying service worker

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Adds a self-destroying service worker
Enabled service woker in deploy command
Changed version to 1.1.14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,11 +28,11 @@
     "deploy:prod": "npm-run-all deploy:prod:*",
     "deploy:prod:build": "NODE_ENV=production npm run build",
     "deploy:prod:clean": "rm -rf .editorconfig .gitignore .github .stylelintrc .vscode stats.json webpack.config.js src LICENSE CONTRIBUTING.md README.md jest",
-    "deploy:prod:pm2": "kill_service_worker=1 pm2 start server --update-env --name=\"sttm-prod\" --node-args=\"-r esm\"",
+    "deploy:prod:pm2": "pm2 start server --update-env --name=\"sttm-prod\" --node-args=\"-r esm\"",
     "deploy:dev": "npm-run-all deploy:dev:*",
     "deploy:dev:build": "npm run deploy:prod:build",
     "deploy:dev:clean": "npm run deploy:prod:clean",
-    "deploy:dev:pm2": "kill_service_worker=1 pm2 start server --update-env --name=\"sttm-dev\" --node-args=\"-r esm\""
+    "deploy:dev:pm2": "pm2 start server --update-env --name=\"sttm-dev\" --node-args=\"-r esm\""
```
Comment (bogas04) [inline package.json]: Instead make it 0 for future use

File: scripts/generate-sw.js [hunk 2]
```diff
@@ -20,4 +30,4 @@ fs.writeFileSync(
       /%CSS_ASSETS/gi,
       `'/assets/css/vendor/foundation.min.css?v=6.2.4', '/assets/css/bundle.css'`
     )
-);
+);*/
```
Comment (bogas04) [inline scripts/generate-sw.js]: Instead have `sourceFile` variable and let the code be the way it was, only change the value of `sourceFile`

File: package.json (hunk 1 above)
Comment (Gauravjeetsingh) [reply package.json]: Done.

File: scripts/generate-sw.js (hunk 2 above)
Comment (Gauravjeetsingh) [reply scripts/generate-sw.js]: Fixed.

**Pull request merged**